catroid/src/main/java/org/catrobat/catroid/visualplacement/VisualPlacementTouchListener.java

```diff
@@ -30,9 +30,13 @@
 public class VisualPlacementTouchListener {
 
 	private Mode mode;
-	private float previousY;
 	private float previousX;
-	private long lastTimeMove = 0;
+	private float previousY;
+	private float oldX;
+	private float oldY;
+	private long touchDownTimestamp = 0;
+	private long lastTimestamp = 0;
+	private long oldTimestamp = 0;
```

Comment:
oldTimeStamp is the one before lastTimeStamp, but we couldn't find a better name. Open for suggestions